ipfix/ipfix.c

```diff
@@ -317,9 +386,17 @@ static clib_error_t * ipfix_init (vlib_main_t * vm)
   /* store this node's vlib_main in the ipfix_main_t */
   sm->vlib_main = vm;
 
+  sm->random_seed = random_default_seed();
+
   /* Initialize configuration values */
-  /* FIXME: don't hardcdoe */
-  sm->exporter_port = 49152;
+  while (1) {
+    rand_port = random_u32(&sm->random_seed);
+    rand_port &= 0xFFFF;
+    if (rand_port >= 49152) {
+      break;
+    }
+  }
```

Comment:
A small little nit, maybe I'm missing something but what do you think of this approach?

```c
// Create random port between 49152 to 0xFFFF
rand_port = (random_u32(&sm->random_seed) % (0xFFFF - 49152)) + 49152
```

Reply:
Thanks for the suggestion! I agree it's nicer, and I added it.